thorlcr/graph/thgraph.cpp

```diff
@@ -2874,6 +2874,11 @@ void CJobBase::startJob()
         else
             IWARNLOG("Failed to capture process stacks: %s", output.str());
     }
+
+    constexpr unsigned defaultNumRenameRetries = 10;
+    unsigned numRenameRetries = getOptInt64("numRenameRetries", getGlobalConfigSP()->getPropInt("expert/@numRenameRetries", defaultNumRenameRetries));
+    if (numRenameRetries)
```

Comment:
If you wanted to disable the retries logic, could zero be an ok value ?

Reply:
If it were 0 in the global configuration, it wouldn't get it here, but it would pick it up/set it the 1st time rename is used in jfile. But agree it's misleading, and means you can't set it to 0 as a workunit or Thor setting.